fuel-core/src/database.rs

```diff
@@ -174,9 +183,21 @@ impl Database {
     }
 
     fn get<V: DeserializeOwned>(&self, key: &[u8], column: ColumnId) -> Result<Option<V>, Error> {
+        #[cfg(feature = "prometheus")]
+        {
+            DATABASE_METRICS.read_meter.inc();
+        }
         self.data
             .get(key, column)?
-            .map(|val| bincode::deserialize(&val).map_err(|_| Error::Codec))
+            .map(|val| {
+                #[cfg(feature = "prometheus")]
+                {
+                    DATABASE_METRICS
+                        .bytes_read_meter
+                        .inc_by(bincode::serialized_size(&val).unwrap_or(0));
```

Comment:
since `val` isn't deserialized yet, we could just get the length directly here:

```suggestion
                        .inc_by(val.len() as u64);
```

Reply:
Was looking for a away to do this, didn't realize I could just call length directly